### PR TITLE
Add inspector panel

### PR DESCRIPTION
**Pull request by lgeiger**
A proper panel for code introspection. See https://github.com/nteract/hydrogen/pull/192

inspector

### CODE REVIEW

**Comment by rgbkrk** [comment]
Dude, awesome.

File: lib/main.coffee [hunk 1]
```diff
@@ -395,30 +397,44 @@ module.exports = Hydrogen =
...
-                        buffer = editor.getBuffer()
-                        buffer.deleteRows(0, buffer.getLineCount())
-                    editor.setSoftWrapped(true)
-                    editor.setLineNumberGutterVisible(false)
-                    editor.insertText(stripAnsi(data['text/plain']))
-                    editor.moveToTop()
-                    atom.workspace.activatePreviousPane()
-                    # Hack for easy closing
-                    editor.save()
+                lines = stripAnsi(data['text/plain']).split('\n')
```
Comment (rgbkrk) [inline lib/main.coffee]: If you want to handle all the data types, check out https://github.com/nteract/transformime.

Comment (lgeiger) [inline lib/main.coffee]: Definitely! This will be the next step :+1:

Comment (rgbkrk) [inline lib/main.coffee]: On a side note, transformime currently has to get all the transforms bundled for a full Jupyter setup: https://github.com/jupyter/jupyter-js-notebook/blob/476fbf8233d2fa87ee0495f769a949eb2a271b67/src/output-area/widget.ts#L121-L136

**Pull request merged**